docker/zookeeper/bin/create-zknode-and-start.sh

```diff
@@ -15,5 +15,14 @@ done
 
 # add default zkNode
 /opt/zookeeper/bin/zkCli.sh -server 127.0.0.1:2181 create /${KILDA_ZKNODE} ""
-
+/opt/zookeeper/bin/zkCli.sh -server 127.0.0.1:2181 create /${KILDA_ZKNODE}/floodlight ""
+/opt/zookeeper/bin/zkCli.sh -server 127.0.0.1:2181 create /${KILDA_ZKNODE}/floodlight/1 ""
+/opt/zookeeper/bin/zkCli.sh -server 127.0.0.1:2181 create /${KILDA_ZKNODE}/floodlight/1/signal "START"
+/opt/zookeeper/bin/zkCli.sh -server 127.0.0.1:2181 create /${KILDA_ZKNODE}/floodlight/1/state ""
```

Comment:
maybe we should set empty state before START? It could be race condition.

1. script sends signal START
2. FL receives this signal, handles it and sets state to 0
3. scrips sets state to ""